src/holon/rule_engine/repositories/repository_base.py

```diff
@@ -62,6 +62,13 @@ def clone(self) -> RepositoryBaseClass:
         """Clone the object"""
         return self.__class__(self.objects[:])
 
+    def join(self, other: RepositoryBaseClass) -> RepositoryBaseClass:
+        """Join two Repositories"""
+        if self.__class__ == other.__class__:
+            return self.__class__(self.objects[:] + other.objects[:])
```

Comment:
Dedupliceren lijkt me hier nodig.

Reply:
Eens, in het specifieke geval gaat het goed, omdat assets of aan een gridconenction of een gridnode hangt. Maar join/merge zou in altijd goed moeten werken